builtin/logical/aws/secret_access_keys.go

```diff
@@ -111,6 +111,48 @@ func (b *backend) secretTokenCreate(s logical.Storage,
 	return resp, nil
 }
 
+func (b *backend) assumeRole(s logical.Storage,
+	displayName, policyName, policy string,
+	lifeTimeInSeconds *int64) (*logical.Response, error) {
```

Comment:
K, other existing STS code.  My golang is too weak today for me to spot oddballs like this.